pkg/jobs/collector_test.go

```diff
@@ -178,71 +178,90 @@ func TestFilterCommands(t *testing.T) {
+			"id":        "CMD-0002",
+			"title":     "kubelet.conf file permissions",
+			"key":       "kubeletConfFilePermissions",
+			"nodeType":  "worker",
+			"audit":     "stat -c %a $kubelet.kubeconfig",
+			"platforms": []interface{}{"k8s", "rke2"},
+		},
+	}
+	commandsMap := map[string][]any{
+		"k8s":  commandsK8s,
+		"rke2": commandsRKE2,
+	}
+
 	tests := []struct {
 		name        string
 		platform    string
 		commandsMap map[string][]any
 		want        *NodeCommands
 	}{
-		{name: "node-collector template",
-			platform: "k8s",
-			commandsMap: map[string][]any{
-				"k8s": {
-					map[string]interface{}{
-						"id":        "CMD-0001",
-						"title":     "kubelet.conf file permissions",
-						"key":       "kubeletConfFilePermissions",
-						"nodeType":  "worker",
-						"audit":     "stat -c %a $kubelet.kubeconfig",
-						"platforms": []interface{}{"k8s", "aks"},
-					},
-					map[string]interface{}{
-						"id":        "CMD-0002",
-						"title":     "kubelet.conf file permissions",
-						"key":       "kubeletConfFilePermissions",
-						"nodeType":  "worker",
-						"audit":     "stat -c %a $kubelet.kubeconfig",
-						"platforms": []interface{}{"k8s", "aks"},
-					},
-				},
-				"aks": {
-					map[string]interface{}{
-						"id":        "CMD-0001",
-						"title":     "kubelet.conf file permissions",
-						"key":       "kubeletConfFilePermissions",
-						"nodeType":  "worker",
-						"audit":     "stat -c %a $kubelet.kubeconfig",
-						"platforms": []interface{}{"k8s", "aks"},
-					},
-					map[string]interface{}{
-						"id":        "CMD-0002",
-						"title":     "kubelet.conf file permissions",
-						"key":       "kubeletConfFilePermissions",
-						"nodeType":  "worker",
-						"audit":     "stat -c %a $kubelet.kubeconfig",
-						"platforms": []interface{}{"k8s", "aks"},
-					},
-				},
+		{
+			name:        "select k8s commands",
+			platform:    "k8s",
+			commandsMap: commandsMap,
+			want: &NodeCommands{
+				Commands: commandsK8s,
 			},
+		},
+		{
+			name:        "select RKE2 commands",
+			platform:    "rke2",
+			commandsMap: commandsMap,
 			want: &NodeCommands{
-				Commands: []any{
-					map[string]interface{}{
-						"id":        "CMD-0001",
-						"title":     "kubelet.conf file permissions",
-						"key":       "kubeletConfFilePermissions",
-						"nodeType":  "worker",
-						"audit":     "stat -c %a $kubelet.kubeconfig",
-						"platforms": []interface{}{"k8s", "aks"},
-					},
-					map[string]interface{}{
-						"id":        "CMD-0002",
-						"title":     "kubelet.conf file permissions",
-						"key":       "kubeletConfFilePermissions",
-						"nodeType":  "worker",
-						"audit":     "stat -c %a $kubelet.kubeconfig",
-						"platforms": []interface{}{"k8s", "aks"},
-					},
-				},
+				Commands: commandsRKE2,
+			},
+		},
+		{
+			name:        "select AKS commands",
+			platform:    "aks",
+			commandsMap: commandsMap,
+			want: &NodeCommands{
+				Commands: commandsK8s,
+			},
```

Comment:
maybe add a comment saying why this gives us the `k8s` commands (because it is unknown). This would help with readability of the tests in the future.

Reply:
Actually, maybe it's better to just not use `aks` altogether as it may cause confusion. What if we just use `unknown` as a platform so it's more obvious that it doesn't exist?

Reply:
sure, `aks` is really bad.
done https://github.com/aquasecurity/trivy-kubernetes/pull/409/commits/3da704c56a8af98a204acbee390d607bdfc08f0e